package.json

```diff
@@ -2,7 +2,7 @@
   "name": "resin-supervisor",
   "version": "1.11.0",
   "scripts": {
-    "postinstall": "bash postinstall.sh",
+    "postinstall": "sh postinstall.sh",
```

Comment:
It is, but we usually add the chmod for things we want to have as executable - it is safer in case, for  instance, someone ever decides to develop on Windows and messes it up in the git tree unknowingly. I like this pattern.